packages/runtime-core/src/components/Teleport.ts

```diff
@@ -467,7 +467,7 @@ function updateCssVars(vnode: VNode) {
   // code path here can assume browser environment.
   const ctx = vnode.ctx
   if (ctx && ctx.ut) {
-    let node = vnode.targetStart
+    let node = vnode.targetStart || (vnode.children as VNode[])[0].el!
```

Comment:
this is not the proper fix. For disabled teleport the begining node should be `vnode.el` and the end node should be `vnode.anchor`.
So the proper fix might be
```ts
let node,anchor
if(isDisabled){
  node=vnode.el
  anchor=vnode.anchor
}else{
  node=vnode.targetStart
  anchor=vnode.targetAnchor
}
while (node && node !== anchor) {
 //...
}
```